src/Tribe/Editor/Warnings.php

```diff
@@ -106,11 +100,19 @@ public function get_recurring_event_warning_message() {
 	 *
 	 * @param string $message Tee message to show.
 	 * @param string $type    Type of message.
+	 * @param string $depends_on Dependency selector.
+	 * @param string $condition Dependency condition like 'checked' | 'not-checked' | 'numeric'.
 	 */
-	public function render_notice( $message, $type = 'info' ) {
-		$icon = 'dashicons-' . $type;
+	public function render_notice( $message, $type = 'info', $depends_on = '', $condition = '' ) {
+		$icon           = 'dashicons-' . $type;
+		$has_dependency = empty( $depends_on ) ? '' : 'tribe-dependent';
+		$classes        = $type . ' ' . $has_dependency;
+		$condition_attr = empty( $condition ) ? '' : 'data-condition-is-' . $condition;
 		?>
-		<div class="ticket-editor-notice <?php echo esc_attr( $type ); ?>">
+		<div class="ticket-editor-notice <?php echo esc_attr( $classes ); ?>"
+		     data-depends="<?php echo esc_attr( $depends_on ); ?>"
```

Comment:
* :warning: Found precision alignment of 1 spaces.
* :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: 2566f955001b262cd1fa736f6e9284a1</sub>